pages/policy.html

```diff
@@ -51,22 +45,32 @@ <h3>Cookies Policy</h3>
+            <p>
+                Cookies are small pieces of data stored in text files that are saved on your computer or other devices when websites are loaded in a browser.
+                They are widely used to remember you and your preferences, either for a single visit (through a “session cookie”) or for multiple repeat visits (using a “persistent cookie”).
+            </p>
+            <p>
+                Session cookies are temporary cookies that are used during the course of your visit to the Website, and they expire when you close the web browser.
+            </p>
+            <p>
+                Persistent cookies are used to remember your preferences within our Website and remain on your desktop or mobile device even after you close your browser or restart your computer.
+                They ensure a consistent and efficient experience for you while visiting the Website and Services.
+            </p>
+            <p>
+                <small>(this text is based on an original template created by websitepolicies.com)</small>
+            </p>
+
             <h3>Cookies on our Website</h3>
-            <p>OpenTTD uses the following cookies on this website, for the following purposes:</p>
-            <ul>
-                <li>Keeping track of customisation wishes of the user on the different websites.</li>
-                <li>Allowing automatic login at different websites.</li>
-            </ul>
-            <h3>Refusing Cookies</h3>
-            <p>Most browsers allow you to refuse to accept cookies.</p>
-            <p>In Internet Explorer, you can refuse all cookies by clicking "Tools", "Internet Options", "Privacy", and selecting 'Block all cookies" using the sliding selector.</p>
-            <p>In Firefox, you can adjust your cookies settings by clicking "Tools", "Options" and "Privacy".</p>
-            <p>Blocking cookies will have a negative impact upon the usability of some websites.</p>
-            <h3>This Cookies Policy</h3>
-            <p>This cookies policy is based on an original template created by <a href="http://template-contracts.co.uk/">template-contracts.co.uk</a> and distributed by <a href="http://freenetlaw.com/">freenetlaw.com</a>.</p>
+            <p>
+                OpenTTD only uses session cookies, and only for the authenticated parts of some of our services (like our wiki, BaNaNaS, etc), so called "necessary cookies".
+                These cookies are only set when you login to those services, and only restrict access to the authenticated part of those services.
+            </p>
+            <p>
+                The name of the cookie always starts with the name of the service (like "wiki" or "bananas") followed by "_sid".
```

Comment:
```suggestion
                OpenTTD only uses session cookies, and only for the authenticated parts of some of our services (like our wiki, BaNaNaS, etc), so called “necessary cookies”.
                These cookies are only set when you login to those services, and only restrict access to the authenticated part of those services.
            </p>
            <p>
                The name of the cookie always starts with the name of the service (like “wiki” or “bananas”) followed by “_sid”.
```

Reply:
I applied the other changes, but not going to do this one :) These kind of quotes are silly and serve no actual purpose other than to be annoying :P Just a `"` works fine :)